ratis-server/src/main/java/org/apache/ratis/server/impl/RaftServerImpl.java

```diff
@@ -86,20 +65,7 @@
 import org.apache.ratis.protocol.SetConfigurationRequest;
 import org.apache.ratis.protocol.SnapshotManagementRequest;
 import org.apache.ratis.protocol.TransferLeadershipRequest;
-import org.apache.ratis.protocol.exceptions.GroupMismatchException;
-import org.apache.ratis.protocol.exceptions.LeaderNotReadyException;
-import org.apache.ratis.protocol.exceptions.LeaderSteppingDownException;
-import org.apache.ratis.protocol.exceptions.NotLeaderException;
-import org.apache.ratis.protocol.exceptions.RaftException;
-import org.apache.ratis.protocol.exceptions.ReadException;
-import org.apache.ratis.protocol.exceptions.ReadIndexException;
-import org.apache.ratis.protocol.exceptions.ReconfigurationInProgressException;
-import org.apache.ratis.protocol.exceptions.ResourceUnavailableException;
-import org.apache.ratis.protocol.exceptions.ServerNotReadyException;
-import org.apache.ratis.protocol.exceptions.SetConfigurationException;
-import org.apache.ratis.protocol.exceptions.StaleReadException;
-import org.apache.ratis.protocol.exceptions.StateMachineException;
-import org.apache.ratis.protocol.exceptions.TransferLeadershipException;
+import org.apache.ratis.protocol.exceptions.*;
```

Comment:
Is there any rule to decide to use * or import individual?
In this PR, you also modified 
```
import org.apache.ratis.proto.RaftProtos.*;
```
to
```
import org.apache.ratis.proto.RaftProtos.DataStreamRequestTypeProto;
import org.apache.ratis.proto.RaftProtos.ForwardRequestTypeProto;
import org.apache.ratis.proto.RaftProtos.MessageStreamRequestTypeProto;
....
```

Reply:
This was to [Get around RaftServerImpl.java File length is 2,019 lines (max allowed is 2,000](https://github.com/apache/ratis/pull/1006/commits/8cc04ea97b5621407814f3aab317ca148769a54f) .  Filed RATIS-1996 to refactor RaftServerImpl.java

Honestly, I feel people focus too much on checkstyle issues.  We are better to put our attention on the code logic instead.